options/locale/locale_en-US.ini

```diff
@@ -489,31 +490,41 @@ keep_email_private_popup = Your email address will be hidden from other users.
 openid_desc = OpenID lets you delegate authentication to an external provider.
 
 manage_ssh_keys = Manage SSH Keys
+manage_ssh_principals = Manage SSH Certificate Principals
 manage_gpg_keys = Manage GPG Keys
 add_key = Add Key
 ssh_desc = These public SSH keys are associated with your account. The corresponding private keys allow full access to your repositories.
+principal_desc = These allowed SSH certificate principals are associated with your account. The corresponding principals allow full access to your repositories.
 gpg_desc = These public GPG keys are associated with your account. Keep your private keys safe as they allow commits to be verified.
 ssh_helper = <strong>Need help?</strong> Have a look at GitHub's guide to <a href="%s">create your own SSH keys</a> or solve <a href="%s">common problems</a> you may encounter using SSH.
 gpg_helper = <strong>Need help?</strong> Have a look at GitHub's guide <a href="%s">about GPG</a>.
 add_new_key = Add SSH Key
 add_new_gpg_key = Add GPG Key
+add_new_principal = Add allowed principal
 ssh_key_been_used = This SSH key has already been added to the server.
 ssh_key_name_used = An SSH key with same name is already added to your account.
+ssh_principal_name_used = A SSH certificate principal with the same name is already added to your account.
```

Comment:
```suggestion
ssh_principal_name_used = A SSH certificate principal with the same name already exists on your account.
```

Reply:
No problem with changing that, but it's verbatim from the line above, shouldn't that change too ?

Reply:
Yeah I would suggest changing the line above too, it just sounds better.